### PR TITLE
Apply updated bootstack template

### PR DESCRIPTION
**Pull request by chanchiwai-ray**
Apply updated [bootstack template: pytest-operator](https://code.launchpad.net/~bootstack-charmers/bootstack-charms-spec/+git/bootstack-charms-spec/+ref/master) at ee3294f3ec03bab8f24599ce3bc208e8fa9261d2.

### CODE REVIEW

**Review by mkalcok — APPROVED** [review]

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  pr:
+    uses: canonical/bootstack-actions/.github/workflows/pull-request.yaml@main
+    secrets: inherit
+    with:
+      python-version-unit: "['3.8', '3.10']"
+      python-version-func: "3.10"
+      tox-version: "<4"
+      commands: "['FUNC_ARGS=\"--series jammy\" make functional']"
```
Comment (rgildein) [inline .github/workflows/pr.yaml]: We had tests for bionic, focal and jammy In previous `func_tests.yaml` workflow, so I think we should stick with that. 
```suggestion
      commands: "['FUNC_ARGS=\"--series bionic\" make functional', 'FUNC_ARGS=\"--series focal\" make functional', 'FUNC_ARGS=\"--series jammy\" make functional']"
```

**Review by rgildein — CHANGES_REQUESTED** [review]

**Review by esunar — APPROVED** [review]
One small [issue](https://github.com/canonical/charm-sosreport/pull/3#discussion_r1149240434) with functional test otherwise LGTM

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (esunar) [reply .github/workflows/pr.yaml]: I agree since we have a lot of bionic clouds

**Review by rgildein — APPROVED** [review]
LGTM

**Pull request merged**